hnn_core/cell.py

```diff
@@ -220,6 +218,64 @@ def _connect(self, gid, gid_dict, pos_dict, p, type_src, name_src,
                     self.parconnect_from_src(
                         gid_src, nc_dict, postsyn))
 
+    # one parreceive function to handle all types of external parreceives
+    def parreceive_ext(self, type, gid, gid_dict, pos_dict, p_ext):
+        """Connect cell to external input."""
+        if self.celltype not in p_ext.keys():
+            return
+
+        gid_src = gid + gid_dict[type][0]
+
+        nc_dict = dict()
+        nc_dict['ampa'] = {
+            'pos_src': pos_dict[type][gid],
+            # index 0 for ampa weight
+            'A_weight': p_ext[self.celltype][0],
+            'A_delay': p_ext[self.celltype][2],  # index 2 for delay
+            'lamtha': p_ext['lamtha'],
```

Comment:
Out of curiosity, what is the `'lamtha'` parameter?

Reply:
it's the spatial decay I think. See [here](https://github.com/jonescompneurolab/hnn-core/blob/master/hnn_core/cell.py#L375-L378)